src/Traits/Enqueue.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Backpack\Base\Traits;
+
+trait Enqueue
+{
+    public static $enqueuedScripts = [];
+    public static $enqueuedStyles = [];
+
+    public static function enqueueAsset($type, $id, $source, $dependencies = [])
+    {
+        switch ($type) {
+            case 'css':
+            case 'style':
+                return self::enqueueStyle($id, $source, $dependencies);
+                break;
+            case 'script':
+            case 'javascript':
+            case 'js':
+                return self::enqueueScript($id, $source, $dependencies);
+                break;
+        }
+    }
+
+    public static function enqueueScript($id, $source, $dependencies)
+    {
+        if (!str_contains($source, '//')) {
+            $path = '/'.ltrim($source, '/');
+            $v = filemtime(realpath(ltrim($path, '/')));
+            $sourceComputed = asset($path.'?v='.$v);
+        } else {
+            $sourceComputed = $source;
+        }
+
+        $sourceObj = (object) [
```

Comment:
Is there any reason to do the double assignment here - it seems just as obvious to do:

```
self::$enqueuedScripts[$id] = (object) [ # ...
```